### PR TITLE
Add Github Action to Build Backend & Frontend

### PR DESCRIPTION
**Pull request by lazuardyk**
## Type of changes

 

Please check the type of change your PR introduces:
- [ ] Bugfix
- [x] Enhancement
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe): 

## What is the current behaviour?
There is no Github action to check the build is successful or not.

## What is the new behavior?
Added GitHub action to prevent error build before merging it to the main branch.

Screenshot (Example error):
Screenshot_1926
Screenshot_1927

For more detail please check here: https://github.com/nodefluxio/weber/actions

### CODE REVIEW

File: .github/workflows/CI.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: "16"
+      - name: "Copy env front-end"
+        run: cp .env.example .env
+        working-directory: ./frontend
+      - name: "Frontend build"
+        run: npm install && npm run build
```
Comment (krismp) [inline .github/workflows/CI.yml:28]: `run: npm install && npm run lint && npm run build`

**Comment by krismp** [comment]
@lazuardyk ada error di frontend [build nya](https://github.com/nodefluxio/weber/runs/4162992843?check_suite_focus=true):

```
./pages/solutions/ekyc.tsx:2:30
Type error: Cannot find module '@/elements/ServiceTitle/ServiceTitle' or its corresponding type declarations.

  1 | import Head from 'next/head'
> 2 | import { ServiceTitle } from '@/elements/ServiceTitle/ServiceTitle'
    |                              ^
  3 | import { getServiceBySlug } from '../../app/api/analyticsAPI'
  4 | import { EkycPage } from '../../app/components/templates/EkycPage/EkycPage'
  ```

**Comment by lazuardyk** [comment]
> @lazuardyk ada error di frontend [build nya](https://github.com/nodefluxio/weber/runs/4162992843?check_suite_focus=true):
> 
> ```
> ./pages/solutions/ekyc.tsx:2:30
> Type error: Cannot find module '@/elements/ServiceTitle/ServiceTitle' or its corresponding type declarations.
> 
>   1 | import Head from 'next/head'
> > 2 | import { ServiceTitle } from '@/elements/ServiceTitle/ServiceTitle'
>     |                              ^
>   3 | import { getServiceBySlug } from '../../app/api/analyticsAPI'
>   4 | import { EkycPage } from '../../app/components/templates/EkycPage/EkycPage'
> ```

Iya lagi gw coba solved, anehnya errornya pas di bagian on ```pull_request```, kalo pas bagian on ```push```nya kaga

**Comment by gregoriusjimmy** [comment]
keren

**Comment by lazuardyk** [comment]
Dah gua solved. menurut gua on ```push``` aja udah cukup jadi gua delete yg on ```pull_request```

> > @lazuardyk ada error di frontend [build nya](https://github.com/nodefluxio/weber/runs/4162992843?check_suite_focus=true):
> > ```
> > ./pages/solutions/ekyc.tsx:2:30
> > Type error: Cannot find module '@/elements/ServiceTitle/ServiceTitle' or its corresponding type declarations.
> > 
> >   1 | import Head from 'next/head'
> > > 2 | import { ServiceTitle } from '@/elements/ServiceTitle/ServiceTitle'
> >     |                              ^
> >   3 | import { getServiceBySlug } from '../../app/api/analyticsAPI'
> >   4 | import { EkycPage } from '../../app/components/templates/EkycPage/EkycPage'
> > ```
> 
> Iya lagi gw coba solve, anehnya errornya pas di bagian on `pull_request`, kalo pas bagian on `push`nya kaga

Solved. menurut gua on ```push``` aja udah cukup gausah pake on pull request

File: .github/workflows/CI.yml (hunk 1 above)
Comment (lazuardyk) [reply .github/workflows/CI.yml]: Done https://github.com/nodefluxio/weber/pull/103/commits/ab6496b3525073031ced6aa26a083d128d8ede5b

Comment (krismp) [reply .github/workflows/CI.yml]: up

Comment (lazuardyk) [reply .github/workflows/CI.yml]: Udah itu ka kwkwk

**Review by krismp — APPROVED** [review]
Looks GOKIL to me

**Review by gregoriusjimmy — APPROVED** [review]

**Pull request merged**